bundles/org.openhab.binding.rotel/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,178 @@
+			<channel id="line2" typeId="frontPanelLine" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="mainZoneType3">
+		<label>Main Zone</label>
+		<description>The controls of the main zone</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="recordSource" typeId="recordSource" />
+			<channel id="dsp" typeId="dsp" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+			<channel id="bass" typeId="bass" />
+			<channel id="treble" typeId="treble" />
+			<channel id="line1" typeId="frontPanelLine" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone2type1">
+		<label>Zone 2</label>
+		<description>The controls of the zone 2</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone2type2">
+		<label>Zone 2</label>
+		<description>The controls of the zone 2</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone3">
+		<label>Zone 3</label>
+		<description>The controls of the zone 3</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone4">
+		<label>Zone 4</label>
+		<description>The controls of the zone 4</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="power">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power ON/OFF the equipment or the zone</description>
+	</channel-type>
+
+	<channel-type id="source">
+		<item-type>String</item-type>
+		<label>Source Input</label>
+		<description>Select the source input</description>
+	</channel-type>
+
+	<channel-type id="recordSource" advanced="true">
+		<item-type>String</item-type>
+		<label>Record Source</label>
+		<description>Select the source to be recorded</description>
+	</channel-type>
+
+	<channel-type id="dsp">
+		<item-type>String</item-type>
+		<label>DSP Mode</label>
+		<description>Select the DSP mode</description>
+	</channel-type>
+
+	<channel-type id="volume">
+		<item-type>Number</item-type>
+		<label>Volume</label>
+		<description>Adjust the volume</description>
+		<state pattern="%d" readOnly="false" />
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+
+	<channel-type id="mute">
```

Comment:
same here, there is a `system.mute` channel type.

Reply:
Done.
I kept the specific channel type (renamed into zoneMute) for the additional zones.

Reply:
Why? You also had used a single channel type before.

Reply:
Previously my label/description was covering the two different "objects" ("the equipment or the zone"). The system channel is fully appropriate for the equipment but less for the zone.
That is the same thing that the inside and outside temperatures when you explained to me in the past that I cannot use the same channel type for both.
But if you prefer, I can the system channel in both cases.

Reply:
Thinking about it, in this condition, I should have created channels types for zone 2, other for zone 3, ... So ok, it could be better to consider the same channel types for the main zone and any other zone, and in this case to use the system channels for zone 2/3/4.
I will change it again.

Reply:
Change done.